aml-development/frontend
# Amlos 890 version number

**Description**     
Place updated version number in a version.txt file. This is done through a TextFileReader that parses the text file and presents it as a ReactJS component. The url for version.txt is:
- http://localhost:8080/#/version

**How to test code**    
pull code and test with version.txt file that is already in the repo. Make sure that file renders properly and that the url path for version.txt works. 

**Please Review**     
@aml-development/aml-backend-team
